pkg/accounts/service/accountVerifyToken.test.ts

```diff
@@ -29,11 +29,7 @@ await accountRepository.create(
     deletedAt: undefined,
   }),
 );
-const mockClock = {
-  now(): bigint {
-    return BigInt(new Date('2023/9/10 00:00:00 UTC').getTime());
-  },
-};
+const mockClock = new MockClock(new Date('2023/9/10 00:00:00 UTC'));
```

Comment:
```suggestion
const mockClock = new MockClock(new Date('2023-09-10T00:00:00Z'));
```
この日付形式は処理系依存なので, ISO 8601 拡張形式にしておきたいです. 他も同様です.